# Candidaalbicans-GenomicChanges

Download and install all packages
https://samtools.github.io/bcftools/

http://www.usadellab.org/cms/?page=trimmomatic

https://sourceforge.net/projects/bio-bwa/files/

https://github.com/lh3/seqtk.git

https://github.com/broadinstitute/pilon/releases/tag/v1.23

https://github.com/vcftools/vcftools

https://tectonic-typesetting.github.io/en-US/install.html 

https://github.com/gabaldonlab/haplotypo 

https://broadinstitute.github.io/picard/


Versions of packages used:

Bcftools v.1.10.2

samtools v.1.10

bwa v.0.7.17

seqtk

Trimmomatic v0.39

Pilon v.1.23

vcftools 0.1.13

Picard v.2.23.8


FASTQ-VCFAnalysis

  -Converting FASTQ files to sam, bam, and VCF files, indexing VCF files, and merging VCF files 
  
  -The prefix ACS is the name for my paired-end sequencing reads 
  
  -CAgenome.fna and ref.fa refer to the reference genome https://www.ncbi.nlm.nih.gov/genome/?term=Candida%20albicans[Organism]&cmd=DetailsSearch
  
  

SNPRelate_PCA: Code to create a PCA plot from combined variant files 

  -ACSoutputv2.vcf.gz combined variant file used to generate PCA plot
  
  -candidaalbicans2021.gds file used in SNPRelate code 
  
  -pop.group file used in SNPRelate code to designate the specific VCF files in the combined vcf along with color designations
  
